vispy/util/logs.py

```diff
@@ -317,14 +317,13 @@ def _handle_exception(ignore_callback_errors, print_callback_errors, obj,
         if this_print == 'full':
             logger.log_exception()
             if exp_type == 'callback':
-                logger.warning("Error invoking callback %s for "
-                               "event: %s" % (cb, event))
+                logger.error("Invoking %s for %s" % (cb, event))
```

Comment:
Because it's actually indicative of an error. I found it weird to have it say `WARNING: Error ...`, seems like a misnomer.